caputilities/enigma.go

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
```

Comment:
change the header to HPE and update licence to 2022

Reply:
changed